### PR TITLE
Document upcoming changes for v2

### PR DESCRIPTION
**Pull request by fuzzybinary**
### What does this PR do? What is the motivation?

Flutter v2 will be out soon with several name changes and capabilities. This updates the documentation to be in line with the new interface.

### Merge instructions
Please wait to merge until v2 is out. I'll comment when we ship v2.

- [x] Please merge after reviewing

### CODE REVIEW

File: content/en/real_user_monitoring/flutter/_index.md [hunk 1]
```diff
@@ -106,19 +106,19 @@ Use the [Datadog Tracking HTTP Client][5] package to enable automatic tracking o
...
   firstPartyHosts: ['example.com'],
 )..enableHttpTracking()
 ```
 
 **Note**: The Datadog Tracking HTTP Client modifies [`HttpOverrides.global`][9]. If you are using your own custom `HttpOverrides`, you may need to inherit from [`DatadogHttpOverrides`][10]. In this case, you do not need to call `enableHttpTracking`. Versions of `datadog_tracking_http_client` >= 1.3 check the value of `HttpOverrides.current` and use this for client creation, so you only need to make sure to initialize `HttpOverrides.global` prior to initializing Datadog.
 
-In order to enable Datadog [Distributed Tracing][6], you must set the `DdSdkConfiguration.firstPartyHosts` property in your configuration object to a domain that supports distributed tracing. You can also modify the sampling rate for distributed tracing by setting the `tracingSamplingRate` on your `RumConfiguration`.
+In order to enable Datadog [Distributed Tracing][6], you must set the `DatadogConfiguration.firstPartyHosts` property in your configuration object to a domain that supports distributed tracing. You can also modify the sampling rate for distributed tracing by setting the `tracingSamplingRate` on your `RumConfiguration`.
```
Comment (0xnm) [inline content/en/real_user_monitoring/flutter/_index.md:117]: I think `RumConfiguration` should be `DatadogRumConfiguration`?

File: content/en/real_user_monitoring/flutter/advanced_configuration.md [hunk 2]
```diff
@@ -327,7 +320,7 @@ Likewise, if you change the value from `TrackingConsent.pending` to `TrackingCon
 
 ## Flutter-specific performance metrics
 
-To enable the collection of Flutter-specific performance metrics, set `reportFlutterPerformance: true` in `RumConfiguration`. Widget build and raster times are displayed in [Mobile Vitals][12]. 
+To enable the collection of Flutter-specific performance metrics, set `reportFlutterPerformance: true` in `RumConfiguration`. Widget build and raster times are displayed in [Mobile Vitals][12].
```
Comment (0xnm) [inline content/en/real_user_monitoring/flutter/advanced_configuration.md:323]:
```suggestion
To enable the collection of Flutter-specific performance metrics, set `reportFlutterPerformance: true` in `DatadogRumConfiguration`. Widget build and raster times are displayed in [Mobile Vitals][12].
```

File: content/en/real_user_monitoring/flutter/_index.md (hunk 1 above)
Comment (fuzzybinary) [reply content/en/real_user_monitoring/flutter/_index.md:117]: Yes, good catch.

**Comment by maycmlee** [comment]
Created [DOCS-6613](https://datadoghq.atlassian.net/browse/DOCS-6613) for docs review.

**Review by jhgilbert — APPROVED** [review]
Approved with some minor changes for style. Thank you!

**Comment by fuzzybinary** [comment]
@jhgilbert Made some changes as a result of a merge from master, can you re-review?

**Review by jhgilbert — APPROVED** [review]
I noticed some incorrectly styled identifiers that I must have missed earlier, so I fixed those. I also removed a stray letter from line 40, please verify that lines 38-40 are now correct before merging: https://github.com/DataDog/documentation/pull/20412/files#diff-831747a3d9061ae6c35df4a94194ab1d2a1a7283ea0a8bea312227fda6d6874bR38

Thanks!

**Comment by fuzzybinary** [comment]
@jhgilbert That all looks good. The errant letter should have been a comma, but it won't break anything not being there.

I added two minor items to the Troubleshooting page but once those are reviewed, this can be merged as v2.0 shipped this morning.

Thanks!

**Pull request merged**